tests/cypress/fixtures/scriptData/e2e/DeveloperFundamentalTestData.json

```diff
@@ -0,0 +1,11 @@
+{
+  "apiCreatorUserRole": "apicreator",
+  "apiPublisherUserRole": "apipublisher",
+  "newUserCreator": "creator",
+  "newUserPublisher": "publisher",
+  "newUserSubscriber": "subscriber",
+  "newUserMike": "mike",
+  "newUserPassword": "password"
+}
+
+
```

Comment:
Lets remove this additional empty lines

Reply:
sure, shall I update this in the next PR?

Reply:
Fixed